soccer/src/soccer/planning/planner/pivot_path_planner.hpp

```diff
@@ -32,6 +32,7 @@ class PivotPathPlanner : public PathPlanner {
         previous_ = Trajectory{};
         cached_pivot_target_ = std::nullopt;
         cached_pivot_point_ = std::nullopt;
+        cached_angle_change_ = std::nullopt;
```

Comment:
same comment here. is this useful?

Reply:
This I believe we need. Otherwise is_done returns true even after reset I believe.